snoop/utils.py

```diff
@@ -1,5 +1,64 @@
 import subprocess
 import re
+import exifread
+from datetime import datetime
+
+def extract_gps_location(tags):
+    def ratio_to_float(ratio):
+        return float(ratio.num) / ratio.den
+
+    def convert(value):
+        d = ratio_to_float(value.values[0])
+        m = ratio_to_float(value.values[1])
+        s = ratio_to_float(value.values[2])
+        return d + (m / 60.0) + (s / 3600.0)
+
+    tags = {key: tags[key] for key in tags.keys() if key.startswith('GPS')}
+
+    lat = tags.get('GPS GPSLatitude')
+    lat_ref = tags.get('GPS GPSLatitudeRef')
+    lon = tags.get('GPS GPSLongitude')
```

Comment:
Can we use `lng` instead of `lon`? Both are valid abbreviations, but `lng` is more commonly used.